src/controllers/login.js

```diff
@@ -0,0 +1,30 @@
+const bcrypt = require('bcryptjs');
+const getData = require('../../src/database/queries/getData');
+
+exports.get = (request, response) => {
+  response.render('login', {
+    js: ['/helpers/collectData', 'login'],
+    css: ['login'],
+    layout: 'login_signup',
+    title: 'Login | Gaza Freelancing Community',
```

Comment:
I think you need a slight change:

.. because we set module.exports, in views/helpers/makeTitle.js
```javascript
const pageTitle = require('./../views/helpers/makeTitle');

exports.get = (request, response) => {
  response.render('login', {
    js: ['helpers/collectData', 'login'],
    css: 'login',
    layout: 'login_signup',
    title: pageTitle.titleName('login'),
});

Reply:
that incorrect

Reply:
@AhmedAl-AlmiI think the view helper should be used in the views.
in the first required it in the app 
then using it in the views like 
```javascript 
<title>pageTitle(title)</title>
```

Reply:
this is not my work,
I just added two rows.
The rest is written by @Angham116

Reply:
yeah, and I am correct that but on another branch.